packages/terra-clinical-item-collection/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Unreleased
 ----------
 ### Changed
 * Update react/react-dom peer-dependency to latest stable minor release
+* Replaced local key code value with keycode-js
```

Comment:
This need to be placed in unreleased section as well.

Reply:
Updated change log. https://github.com/cerner/terra-clinical/pull/479/commits/17b14e8a1d8ee71d2c742119ed6afea9825731bc